server/sonar-server/src/main/java/org/sonar/server/computation/AnalysisReportService.java

```diff
@@ -130,9 +133,10 @@ private DefaultIssue setFieldDiffs(DefaultIssue issue, String diffFields, Date a
     return issue;
   }
 
-  private DefaultIssue setComponentId(DefaultIssue issue, ReportComponent component) {
+  private DefaultIssue setComponent(DefaultIssue issue, @Nullable ReportComponent component) {
```

Comment:
exact. I kept this nullable parameter because it was done before :-) 
I drop it.